src/List/list.store.js

```diff
@@ -65,30 +89,55 @@ export default Store.create({
         this.listSourceSubject.onNext(Observable.fromPromise(this.state.pager.getPreviousPage()));
     },
 
-    async filter(modelType, searchString, searchBy, canManage, complete, error) {
+    async filter(modelType, searchString, filterByRole, filterByGroup, canManage, complete, error) {
         getD2().then(d2 => {
             if (!d2.models[modelType]) {
                 error(`${modelType} is not a valid schema name`);
             }
-
             let modelDefinition;
             if (searchString) {
-
-                if (searchBy === 'name') {
+                if (filterByRole && filterByGroup) {
```

Comment:
@adrianq I have tried couple of ways to chain the filtering query, but I couldn't get the filter query to work properly, do you think this filtering syntax has problem?